articles/machine-learning/how-to-manage-compute-instance.md

```diff
@@ -159,7 +159,7 @@ For each compute instance in a workspace that you created (or that was created f
 ---
 
 > [!Caution]
-> Applying resource locks, such as "Delete" or "Read-only", to the resource group containing your compute instances can prevent operations like creation, resizing, or deletion of these instances. Ensure that resource locks are configured appropriately to avoid unintended disruptions. 
+> Applying resource locks, such as "Delete" or "Read-only", to the resource group containing your Machine Learning workspace or to a separate resource group where you've configured a Virtual Network (VNet), can prevent operations like creation, resizing, or deletion of these instances. Ensure that resource locks are configured appropriately to avoid unintended disruptions. 
```

Comment:
```suggestion
> Applying resource locks, such as "Delete" or "Read-only", to the resource group that contains your Machine Learning workspace or to a separate resource group where you've configured a virtual network can prevent operations like creation, resizing, or deletion of these instances. Ensure that resource locks are configured appropriately to avoid unintended disruptions. 
```
Edits to align with Microsoft style